src/__tests__/index.ts

```diff
@@ -192,6 +238,20 @@ describe('verifyPresentation', () => {
     expect(verified.verifiablePresentation).toBeDefined()
   })
 
+  it('rejects a Presentation without matching challenge', () => {
+    const options: VerifyPresentationOptions = {
```

Comment:
It's a good idea to tell jest how many assertions are made whenever you're dealing with async code in a test.
Otherwise, there is a risk that the test will pass before the promise has time to be rejected/resolved.

```suggestion
    expect.assertions(1)
    const options: VerifyPresentationOptions = {
```

Reply:
I can add those.